application/Config/Services.php

```diff
@@ -530,6 +530,21 @@ public static function viewcell($getShared = true)
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * The Typography class provides a way to format text in semantically relevant ways.
+	 */
+	public static function typography($uri = null, $getShared = true)
```

Comment:
We don't need the $uri parameter since it's not a dependency for the typography class.